public/javascripts/usage_bar_chart.js

```diff
@@ -44,15 +46,16 @@
       } else if (format === "months") {
         var domain = [startDate, endDate];
         var length = d3.time.months(startDate, endDate).length;
-        width = 400;
+        width = 840;
```

Comment:
@kjgarza does it make sense to have a fixed width of the whole chart if the width of each bar is fixed? Shouldn't the width rather be calculated, depending on the number of months?

Reply:
That's another way to do it. 
I was thinking the main problem is screen size. The panel box has a maximum of 1017px (not matter the screen). But the maximum chart size I have been able to fit in a standard size screen. its 840.

Over the last 6 months, only 1.37% of the visits used a screen resolution lower than 1024 px on a desktop.
        
https://analytics.google.com/analytics/web/#/report/visitors-mobile-overview/a22806196w102169300p106174218/_u.date00=20181101&_u.date01=20190324&explorer-table.secSegmentId=analytics.screenResolution&explorer-table.plotKeys=%5B%5D

Reply:
You could have a fixed bar width with a maximal width of 840px.

Reply:
fix 89baad49981f685e9a80612ac5b3065b5e7539c5